67_RayQueryGeometry/app_resources/render.comp.hlsl

```diff
@@ -2,6 +2,7 @@
 
 struct SPushConstants
 {
+    bool useIndex;
     uint64_t vertexBufferAddress;
     uint64_t indexBufferAddress;
```

Comment:
don't use `bool`s passing data around, you can e.g. check that `indexBufferAddress!=vertexBufferAddress` to see if you have an index buffer instead